documentation/commands/README.md

```diff
@@ -1,3 +1,475 @@
+
+where `PRIORITY` can be `research`, `standard`, `priority`, or `express`.
+
+In case that the sample has been downsampled, provide how many reads it has been downsampled to.
+
+```$ cg add sample --downsampled INTEGER CUSTOMER_ID SAMPLE_NAME```
+
+### cg add user
+Add a new user with an EMAIL (login) and a NAME (full). ```$ cg add user EMAIL NAME```
+
+It is required to connect the user to a customer. ```$ cg add user --customer CUSTOMER_ID EMAIL NAME```
+
+To give a user admin rights, use the `--admin` flag. ```$ cg add user --admin EMAIL NAME```
+
+## cg analysis
+Prepare and start a MIP analysis for a case. To do this use
+
+```$ cg analysis --family FAMILY_ID```
+
+where `FAMILY_ID` is the case name, e.g. `cuddlygull`.
+
+Using this option will overwrite any configuration files, so if changes has been made to these that should be kept, use instead `cg analysis start`.
+
+To change the quality of service for an analysis use
+
+```$ cg analysis --family FAMILY_ID --priority PRORITY```
+
+where `PRIORITY` could be `low`, `normal`, or `high`. The default is set in statusDB.
+
+Notification of errors will by default be sent to the email of the user who started the analysis. To change the email use
+
+```$ cg analysis --family FAMILY_ID --email TEXT```
+
+where `TEXT` is the email address.
+
+### cg analysis auto
+Start all analyses that are ready for analysis.
+
+### cg analysis config
+Generate a configuration file (pedigree.yaml) for an analysis.
+
+```$ cg analysis config FAMILY_ID```
+
+To print the config to console, rather then creating the `pedigree.yaml`, use the `--dry` flag:
+
+```$ cg analysis config --dry FAMILY_ID```
+
+### cg analysis link
+Link FASTQ files for a sample.
+
+```$ cg analysis SAMPLE_ID```
+
+To link files for an entire case, use the `--family` flag:
+
+```$ cg analysis link --family FAMILY_ID```
+
+### cg analysis panel
+Write aggregated gene panel file.
+
+```$ cg analysis panel FAMILY_ID```
+
+To only print the panel to console, rather than saving it to a file, use the `--print` flag:
+
+```$ cg analysis panel --print FAMILY_ID```
+
+### cg analysis start
+Start the analysis pipeline for a family.
+
+```$ cg analysis start FAMILY_ID```
+
+To change the quality of servive for an analysis use
+
+```$ cg analysis start --priority PRIORITY FAMILY_ID```
+
+where `PRIORITY` could be `low`, `normal`, or `high`. The default is set in statusDB.
+
+## cg backup
+Backup utilities.
+
+### cg backup fetch-flowcell
+Fetch the first flowcell in the requested queue from backup.
+
+```$ cg backup fetch-flowcell --flowcell FLOWCELL_ID```
+
+## cg clean
+Remove stuff. *Use caution.*
+
+### cg clean beacon
+Remove beacon for a sample or one or more affected samples from a family. It is required to specify what item type that should be removed. Example:
+
+```$ cg clean beacon --item_type TYPE ITEM_ID```
+
+Where `TYPE` can be `family` or `sample`. cleans the beacon database of the inserted variants
```

Comment:
'cleans the beacon database of the inserted variants', could be set under the title.